src/frontend/src/components/Header/index.tsx

```diff
@@ -7,10 +7,12 @@ import { connect } from 'react-redux';
 
 interface Props {
   isAuthenticated: boolean;
+  isAdmin: boolean; // Added 'isAdmin' for conditional rendering to work (line 35/36)
```

Comment:
Comments like these become quickly outdated (especially line numbers). The commit message is probably a better place for this comment.